### ISSUE TITLE
error msg

### ISSUE THREAD

**Issue by jiohb** [no0be/DNSlivery#7]
```bash
➜  DNSlivery git:(master) ✗ python3.6 dnslivery.py eth0 dnsd.fuy.com vps.fuy.com

DNSlivery - Easy files and payloads delivery over DNS

[*] File "nidaye" ready for delivery at nidaye.dnsd.fuy.com (1 chunks)
[*] File "requirements.txt" ready for delivery at requirements-txt.dnsd.fuy.com (1 chunks)
[*] File "dnslivery.py" ready for delivery at dnslivery-py.dnsd.fuy.com (46 chunks)
[*] File "LICENSE" ready for delivery at license.dnsd.fuy.com (6 chunks)
[*] File "README.md" ready for delivery at readme-md.dnsd.fuy.com (38 chunks)
[*] File ".gitignore" ready for delivery at -gitignore.dnsd.fuy.com (1 chunks)
[*] Listening for DNS queries...

match existing file!
[+] Delivering nidaye print launcher to 58.217.249.133
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/scapy/compat.py", line 117, in raw
    return bytes(x)
  File "/usr/local/lib/python3.6/site-packages/scapy/packet.py", line 441, in __bytes__
    return self.build()
  File "/usr/local/lib/python3.6/site-packages/scapy/packet.py", line 556, in build
    p = self.do_build()
  File "/usr/local/lib/python3.6/site-packages/scapy/packet.py", line 538, in do_build
    pkt = self.self_build()
  File "/usr/local/lib/python3.6/site-packages/scapy/packet.py", line 519, in self_build
    p = f.addfield(self, p, val)
  File "/usr/local/lib/python3.6/site-packages/scapy/fields.py", line 134, in addfield
    return s + struct.pack(self.fmt, self.i2m(pkt, val))
  File "/usr/local/lib/python3.6/site-packages/scapy/layers/dns.py", line 371, in i2m
    x = len(rdataf.i2m(pkt, pkt.rdata))
  File "/usr/local/lib/python3.6/site-packages/scapy/layers/dns.py", line 351, in i2m
    while len(text) >= 255:
TypeError: object of type 'int' has no len()

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/scapy/compat.py", line 117, in raw
    return bytes(x)
  File "/usr/local/lib/python3.6/site-packages/scapy/packet.py", line 441, in __bytes__
    return self.build()
  File "/usr/local/lib/python3.6/site-packages/scapy/packet.py", line 556, in build
    p = self.do_build()
  File "/usr/local/lib/python3.6/site-packages/scapy/packet.py", line 541, in do_build
    pay = self.do_build_payload()
  File "/usr/local/lib/python3.6/site-packages/scapy/packet.py", line 528, in do_build_payload
    return self.payload.do_build()
  File "/usr/local/lib/python3.6/site-packages/scapy/packet.py", line 541, in do_build
    pay = self.do_build_payload()
  File "/usr/local/lib/python3.6/site-packages/scapy/packet.py", line 528, in do_build_payload
    return self.payload.do_build()
  File "/usr/local/lib/python3.6/site-packages/scapy/packet.py", line 541, in do_build
    pay = self.do_build_payload()
  File "/usr/local/lib/python3.6/site-packages/scapy/packet.py", line 528, in do_build_payload
    return self.payload.do_build()
  File "/usr/local/lib/python3.6/site-packages/scapy/packet.py", line 538, in do_build
    pkt = self.self_build()
  File "/usr/local/lib/python3.6/site-packages/scapy/packet.py", line 519, in self_build
    p = f.addfield(self, p, val)
  File "/usr/local/lib/python3.6/site-packages/scapy/fields.py", line 905, in addfield
    return s + self.i2m(pkt, val)
  File "/usr/local/lib/python3.6/site-packages/scapy/layers/dns.py", line 255, in i2m
    return raw(x)
  File "/usr/local/lib/python3.6/site-packages/scapy/compat.py", line 119, in raw
    return bytes(x, encoding="utf8")
TypeError: encoding without a string argument

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "dnslivery.py", line 184, in <module>
    while True: dns_listener = sniff(filter='udp dst port 53', iface=args.interface, prn=dns_handler)
  File "/usr/local/lib/python3.6/site-packages/scapy/sendrecv.py", line 886, in sniff
    r = prn(p)
  File "dnslivery.py", line 97, in dns_handler
    send(response_pkt, verbose=0, iface=args.interface)
  File "/usr/local/lib/python3.6/site-packages/scapy/sendrecv.py", line 314, in send
    realtime=realtime, return_packets=return_packets)
  File "/usr/local/lib/python3.6/site-packages/scapy/sendrecv.py", line 287, in __gen_send
    s.send(p)
  File "/usr/local/lib/python3.6/site-packages/scapy/arch/linux.py", line 533, in send
    sx = raw(ll(x))
  File "/usr/local/lib/python3.6/site-packages/scapy/compat.py", line 119, in raw
    return bytes(x, encoding="utf8")
TypeError: encoding without a string argument

```
file content:

```bash
➜  DNSlivery git:(master) ✗ cat nidaye 
zheshizenmehuishi?

```

**Comment by no0be** [comment]
What version of scapy are you using?
```
pip list | grep scapy
```

DNSlivery does not work with the latest 2.4.2 and you need to make sure you're using 2.4.0 until I find the time to figure out what's going wrong with the latest one.

**Comment by jiohb** [comment]
thank you very much. 
It'a all my fault, I use scapy2.4.2.

**Comment by no0be** [comment]
No worries, I need to fix the code to work with scapy 2.4.2.
